logstash-core/src/test/java/org/logstash/AccessorsTest.java

```diff
@@ -261,14 +257,4 @@ public void testListIndexOutOfBounds(int i) {
       }
     }
 
-    public static class TestListIndex {
-      public void testListIndexOutOfBounds() {
```

Comment:
This looks like it may have been an attempt to test these cases.  What happens if you just pull the test method out of the inner class and slap a @Test annotation on it ?